src/main/app/fileUpload/documentManager.ts

```diff
@@ -12,6 +12,7 @@ export const uploadDocument = async (formData, s2sToken, req) => {
     authorization: `Bearer ${req.session.user.accessToken}`,
     serviceAuthorization: s2sToken,
   };
+  console.log("Document Upload: Upload start");
```

Comment:
Can you create a logger instead please as a constant at the top of the file below the imports:
const logger = Logger.getLogger('documentManager');

the replace the console log with:
logger.info('Document Upload: Upload start');